### PR TITLE
VACMS-18723 Exported and added configs to git for Mission Explainer

### PR DESCRIPTION
**Pull request by jv-agile6**
## Description
Added a Mission Explainer content block to the Vet center content type.
Relates to #18723
### Centralized Content
Screenshot 2024-08-07 at 9 14 15 AM
Screenshot 2024-08-09 at 9 08 15 AM

### Vet Center Content Type
Screenshot 2024-08-09 at 12 49 24 PM

## QA steps

As a drupal admin:
### Validate that a centralized content descriptor content block was added to VetCenter Centralized Content
- [x] Visit https://pr18890-tt9t9v1nunrmm5opymrkq5ncspqnmuxa.ci.cms.va.gov/centralized-content/national-vet-center-content
- [x] Validate this block has two fields, One for the title and one for the rich text field
- [x] content block is populated with the following copy: 

> "Our services
> We offer these confidential services at our Vet Centers:
> 
> Counseling and a range of other services for individuals, families, and group sessions
> Alternative therapies and events may be offered such as art, equine, yoga, or fly fishing
> Connections to more support in VA and your community
> Note: You may be eligible for these services no matter your discharge status or eligibility for VA health care. And we won’t share anything about you or the services you receive at a Vet Center without your permission."

- [x] Content block shows on node:edit and node:view(if low LOE) on VetCenter Content Type

### Update Nodes by performing a bulk edit to the centralized content 

- [x] Go to the bulk edit page [filtered by vet center content type](https://pr18890-tt9t9v1nunrmm5opymrkq5ncspqnmuxa.ci.cms.va.gov/admin/content/bulk?title=&type=vet_center&moderation_state=All&taxonomy_entity_index_tid_depth=All)
- [x] Select all results
- [x] Choose modify field values, and apply
- [x] Choose Mission Explainer field
- [x] Apply and confirm

- [x] Then check a [Vet Center](https://pr18890-tt9t9v1nunrmm5opymrkq5ncspqnmuxa.ci.cms.va.gov/duluth-vet-center) node for Mission Explainer on node view.

### Select Team for PR review

- [ ] `CMS Team`
- [ ] `Public websites`
- [x] `Facilities`
- [ ] `User support`
- [ ] `Accelerated Publishing`

### CODE REVIEW

**Comment by omahane** [comment]
@davidmpickett @laflannery Could you take a look at this before I look at the code/config: https://pr18890-tt9t9v1nunrmm5opymrkq5ncspqnmuxa.ci.cms.va.gov/admin/content

**Comment by davidmpickett** [comment]
Looking good! @jv-agile6 Some comments

1. Centralized content - Please drag the two paragraphs for Mission Explainer to be above Vet Center non-traditional hours. Our general principle is that elements in the Drupal UI should be in the same order as the items will appear on the corresponding FE page to help reduce cognitive load for editors.
2. `Centralized content descriptor` copy suggestions:
Title: Vet Center services overview
Body: This content appears in a blue box at the top of Vet Center pages.
3. EFF box on vet center nodes - Position should probably be under `Top of page information` rather than above it (same principle as #1)
4. EFF box on Vet center nodes - change label from Mission explainer to "Vet Center services overview" 
5. EFF box on Vet center nodes - probably needs a tool tip with the standard "why can't I edit this?" text

**Comment by laflannery** [comment]
Can we add an anchor link on the Vet Center edit screen under "Jump to edit" for this section? I'm not sure how difficult this is but these are all out of whack so if we start to fix them little by little it would help.

**Comment by jv-agile6** [comment]
🤔 Is this a green 🟢  to merge? 🚂 It's got all the right approval, 🥇  right? @omahane

**Comment by davidmpickett** [comment]
@jv-agile6 I don't see any ✅ official approvals on the PR. While @omahane is out, you may need @dsasser to step in to review and approve.

**Comment by jilladams** [comment]
@laflannery I think it would be better since Jerry is spinning several plates / this PR has been in motion for awhile if you're willing to ticket so we can come back to fixing these holistically.

**Comment by omahane** [comment]
> Can we add an anchor link on the Vet Center edit screen under "Jump to edit" for this section? I'm not sure how difficult this is but these are all out of whack so if we start to fix them little by little it would help.

@laflannery I'm not sure why would we do that, as this is not content that an editor can edit.

**Comment by omahane** [comment]
@davidmpickett (cc: @jilladams @jv-agile6 
The prod centralized content has been updated:
https://prod.cms.va.gov/centralized-content/national-vet-center-content

The PR Env has been updated to match the suggested changes:
## Node:view
Screenshot 2024-08-15 at 12 28 16 AM

## Node:edit
Screenshot 2024-08-15 at 12 29 15 AM

**Comment by dsasser** [comment]
@jv-agile6 the QA steps here are hard to follow from a numbering standpoint, and the steps lack links. Can you take a pass at updating the steps to be easier to follow for someone not familiar with this work?

**Review by dsasser — APPROVED** [review]
Changes look good and QA steps passed nice work ✅

**Comment by jv-agile6** [comment]
Approved @dsasser
